patch-package
--- a/node_modules/@babel/core/lib/config/config-chain.js
+++ b/node_modules/@babel/core/lib/config/config-chain.js
@@ -317,11 +317,15 @@ function normalizeOptions(opts) {
   const options = Object.assign({}, opts);
   delete options.extends;
   delete options.env;
+  delete options.overrides;
   delete options.plugins;
   delete options.presets;
   delete options.passPerPreset;
   delete options.ignore;
   delete options.only;
+  delete options.test;
+  delete options.include;
+  delete options.exclude;
 
   if (options.sourceMap) {
     options.sourceMaps = options.sourceMap;
