patch-package
--- a/node_modules/babel-loader/lib/index.js
+++ b/node_modules/babel-loader/lib/index.js
@@ -79,12 +79,12 @@ function _loader() {
       throw new Error(`babel-loader ^8.0.0-beta.3 requires @babel/core@7.0.0-beta.41, but ` + `you appear to be using "${babel.version}". Either update your ` + `@babel/core version, or pin you babel-loader version to 8.0.0-beta.2`);
     }
 
-    const config = babel.loadPartialConfig(programmaticOptions);
+    const config = programmaticOptions.configFile ? { options: programmaticOptions } : babel.loadPartialConfig(programmaticOptions);
 
     if (config) {
       let options = config.options;
 
-      if (overrides && overrides.config) {
+      if (overrides && overrides.config && !programmaticOptions.configFile) {
         options = yield overrides.config.call(this, config, {
           source,
           customOptions
