diff --git a/node_modules/braft-convert/dist/index.js b/node_modules/braft-convert/dist/index.js
index cfd9457..8e792c5 100644
--- a/node_modules/braft-convert/dist/index.js
+++ b/node_modules/braft-convert/dist/index.js
@@ -26,6 +26,11 @@ var convertRawToHTML = exports.convertRawToHTML = function convertRawToHTML(rawC
     options.contentState = contentState;
     return (0, _draftConvert.convertToHTML)((0, _configs.getToHTMLConfig)(options))(contentState);
   } catch (error) {
+    console.error(JSON.stringify({
+      ...rawContent,
+      severity: 'ERROR',
+      message: 'convertRawToHTML',
+    }));
     console.warn(error);
     return '';
   }
