patch-package
--- a/node_modules/rc-form/lib/createDOMForm.js
+++ b/node_modules/rc-form/lib/createDOMForm.js
@@ -101,7 +101,8 @@ var mixin = {
               if (instance) {
                 var node = _reactDom2['default'].findDOMNode(instance);
                 var top = node.getBoundingClientRect().top;
-                if (firstTop === undefined || firstTop > top) {
+                // FIXME
+                if (node.type !== 'hidden' && (firstTop === undefined || firstTop > top)) {
                   firstTop = top;
                   firstNode = node;
                 }
