patch-package
--- a/node_modules/koa-better-http-proxy/app/steps/sendProxyRequest.js
+++ b/node_modules/koa-better-http-proxy/app/steps/sendProxyRequest.js
@@ -8,12 +8,25 @@ function sendProxyRequest(Container) {
   var reqOpt = Container.proxy.reqBuilder;
   var options = Container.options;

+  if (!process.env.TEST_PROD_SERVER) throw new Error('this patch should run in `test-prod-server`');
+  const [newApi, newPort = 80] = (process.env.TEST_API || 'api.stage.meepcloud.com').split(/:/);
+
+  reqOpt.port = newPort;
+  reqOpt.path = reqOpt.path.replace(/meepshop-api:15265/, newApi);
+  reqOpt.headers['x-meepshop-domain'] = process.env.TEST_DOMAIN || 'bellatest.stage.meepcloud.com';
+  reqOpt.host = newApi;
+
   return new Promise(function(resolve, reject) {
     var protocol = Container.proxy.requestModule;
     var proxyReq = protocol.request(reqOpt, function(rsp) {
       var chunks = [];
       rsp.on('data', function(chunk) { chunks.push(chunk); });
       rsp.on('end', function() {
+        if (/auth\/login$/.test(reqOpt.path))
+          process.env.AUTH_TOKEN = rsp.headers['x-meepshop-authorization-token'];
+        else if (/auth\/signout/.test(reqOpt.pah))
+          process.env.AUTH_TOKEN = false;
+
         Container.proxy.res = rsp;
         Container.proxy.resData = Buffer.concat(chunks, chunkLength(chunks));
         resolve(Container);
